src/list/App.vue

```diff
@@ -373,7 +369,14 @@ export default {
   },
   created() {
     this.onlyNew = new URL(window.location.href).searchParams.get('only-new') === 'true'
+    const options = [
+      { title: this.$t('message.yes'), value: true },
+      { title: this.$t('message.no'), value: false }
+    ]
+    this.optimizations = options
+    this.isNewOptions = options
     this.init()
+    chrome.extension.getBackgroundPage().console.log(options, 'lang')
```

Comment:
should be as below suggestion because of web extension's polyfill

```suggestion
    browser.extension.getBackgroundPage().console.log(options, 'lang')
```